static/styles/subscriptions.css

```diff
@@ -10,77 +10,18 @@
     margin: 0;
 }
 
-.subscription-email-hint-image {
-    float: right;
-    width: 80px;
-}
-
 .subscription_header.collapsed {
     background-color: inherit;
 }
 
-.color_swatch {
-    display: inline-block;
-    height: 18px;
-    width: 18px;
-    padding: 0;
-    margin: 0 0 0 10px;
-    vertical-align: middle;
-}
-
 .subscriptions div #response {
     overflow-wrap: break-word;
 }
 
-.subscription-info-container {
-    display: inline-block;
-    /* subtract out the width of the subscribe button and arrow. */
-    width: calc(100% - 260px);
-    /* to center the container vertically. */
-    margin-top: 13px;
-}
-
-.subscription-setting-icon {
-    vertical-align: top;
-    display: inline-block;
-}
-
-.subscription-info {
-    display: inline-block;
-    /* to provide space for the setting-icon. */
-    max-width: calc(100% - 31px);
-    margin-top: 1px;
-    overflow: hidden;
-    word-wrap: normal;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-}
-
-.subscription-info .light {
-    font-weight: 300;
-}
-
 .subscription_header {
     min-height: 47px;
 }
 
-.subscription-name-row {
-    display: inline-block;
-}
-
-.subscription-name-row,
-.subscription-description-row {
-    vertical-align: top;
-}
-
-.subscription_header .subscription_lock {
-    font-size: 23px;
-    display: inline-block;
-    vertical-align: middle;
-    margin-left: 12px;
-    margin-right: 1px;
-}
```

Comment:
Hmm this seems like it has some JS associated with it.

```
$ git grep subscription_header
static/js/ui_init.js:    $("#manage_streams_container").on("mouseover", ".subscription_header", function () {
static/js/ui_init.js:    $("#manage_streams_container").on("mouseout", ".subscription_header", function () {
...
```

Reply:
(Possibly we get to delete that too).

Reply:
`subscription_header` does, but `subscription_lock` does not.